packages/template-retail-react-app/app/components/product-view/index.jsx

```diff
@@ -524,6 +527,7 @@ ProductView.propTypes = {
     isProductPartOfSet: PropTypes.bool,
     category: PropTypes.array,
     isProductLoading: PropTypes.bool,
+    isBasketLoading: PropTypes.bool,
```

Comment:
I feel slightly wary of the increased complexity of adding new props, but I don't have any better alternatives, so maybe this is just a useless comment...